### ISSUE TITLE
Wire up some autoformatter (`black`?)

### ISSUE THREAD

**Issue by PawelLipski** [VirtusLab/git-machete#140]
A check will be necessary in the CI pipeline.

**Comment by rifel123** [comment]
Here you can find version of cmd.py after black formatting: [black diff](https://github.com/VirtusLab/git-machete/commit/4364b46cdd1a9945e5037d22b71bd88c18a8f3b0)
The output above is performed with standard line length described in PEP8 (88 characters?), so most of changes are line breaks and rest is replacing `'` into `"`. 

Let's set line length to 300: [black diff with line length 300](https://github.com/VirtusLab/git-machete/commit/1ae6d58f27c59e664efefb1c8e45da78680b7e40)

And right now most of changes are line merges.

In our ci process we would not want to insert changes inplace, but instead run with --check flag which outputs non zero exit code when at least one file needs to be changed, here is example from my local run:
<img width="1135" alt="Screenshot 2021-08-31 at 09 09 38" src="https://user-images.githubusercontent.com/73847352/131458765-84e30dd6-c80c-4f7c-9639-b4265dfa1849.png">

Basically `Black` does same thing to `Flake8` but also can apply it's finding into code. On the other hand `Black` blindly follows PEP8 rules, and there is no way to ignore  certain errors (only way is to wrap with ` # fmt: off/on` certain block of code to skip checks)

**Comment by PawelLipski** [comment]
Wow I don't know if I like certain decisions made by `black`...

![image](https://user-images.githubusercontent.com/3383210/131521451-78f84a38-475d-4dff-a860-109c28e2e30b.png)

Extracting type params to a separate line seems like an overkill 😅 are things like that configurable in `black`?

**Comment by PawelLipski** [comment]
Similarly for short function params like

![image](https://user-images.githubusercontent.com/3383210/131521747-a8c01b5d-3dbc-4fc7-a75f-56909efd775e.png)

**Comment by PawelLipski** [comment]
Haha oh lol 😂 

![image](https://user-images.githubusercontent.com/3383210/131521848-9892807e-5075-4e91-8068-ca5cac347a7c.png)

**Comment by PawelLipski** [comment]
Yeah, the tool generally seems to make statements spread across more lines than needed for readability :/ at least with the current (default?) config

**Comment by rifel123** [comment]
Yeah but `black` is not really configurable. Take it or leave it. The only things which are configurable are line length,                 --skip-string-normalization -> Don't normalize string quotes or prefixes,
--skip-magic-trailing-comma -> Don't use trailing commas as a reason to split lines.

user cannot any specify any rules about formatting itself because it's taken directly from PEP8.

**Comment by PawelLipski** [comment]
Okay... if so then I'm against applying `black`, looks too far-fetched/too opinionated for me ☹️

**Comment by PawelLipski** [comment]
No further comments here... so lemme consider this closed 🚫
